state/index.js

```diff
@@ -10,10 +11,20 @@ const toDict = (array, keyFn) => array.reduce((obj, item) => ({
   [keyFn(item)]: item,
 }), {});
 
-const getSiteState = () => {
-  return getJobs(fetch, process.env.WORKABLE_API_KEY).then(jobs =>
-    ({ jobs, job: toDict(jobs, j => j.slug), ...state })
-  );
-};
+const getSiteState = () => (new Promise(res => {
+  const state = {
+    ...initialState,
+  };
+
+  Promise.all([
```

Comment:
`Bluebird.props` was used here before and made it slightly more idiomatic than mutating `state`